app/src/components/features/rules/RuleBuilder/Body/RuleBuilderBody.css

```diff
@@ -18,3 +19,28 @@
   display: flex;
   justify-content: center;
 }
+
+.sample-rule-card-body {
+  opacity: 0.6;
+}
+
+.sample-rule-overlay {
```

Comment:
Lets keep the overlay at top instead of middle, for some rules it goes below the fold
<img width="1440" alt="Screenshot 2024-09-25 at 12 19 50 AM" src="https://github.com/user-attachments/assets/4ed57ad8-0dd9-4397-8e92-553f8b9e7a53">